asciidoc-confluence-publisher-doc/etc/docs/00-index.adoc

```diff
@@ -251,6 +251,8 @@ The Docker image for the code on master is automatically published as version `0
 Instead of providing the `asciidocRootFolder` configuration property as with the Maven plugin, the AsciiDoc sources
 have to be mounted to the `/var/asciidoc-root-folder` volume.
 
+If you are not able or allowed to mount anything into this image you can change this path by providing the environment variable `ASCIIDOC_ROOT_FOLDER`.
```

Comment:
Isn't `not able or allowed` a bit misleading? If I understood correctly, in your scenario, there is simply no need to mount anything into the container, as the documentation sources are getting checked out into the Confluence Publisher docker container (as you are using that docker image as the build environment). In this case, you only want to point the environment variable to your already existing documentation sources, right?

So maybe describing this as "in case your documentation sources are already available under a different path in your Docker container, you can specify the path to the documentation sources via ..."

Reply:
I liked your phrasing and change the paragraph with c00206a